test/orbit_web/controllers/sign_in_controller_test.exs

```diff
@@ -7,6 +7,74 @@ defmodule OrbitWeb.SignInControllerTest do
 
   import Ecto.Query
 
+  describe "index" do
+    @tag :authenticated
```

Comment:
**suggestion:** it's probably worth having a test (both for `index` and also `submit`) making sure that unauthenticated connections receive the appropriate response code, and don't get any sensitive data.

Reply:
7e57dd0

Reply:
(and 53bf3b0)